.github/labeler.yml

```diff
@@ -9,3 +9,7 @@ documentation:
 github structure:
   - any: [".github/**/*"]
     all: ["!**/*.py"]
+
+translation:
+  - any: ["custom_components/google_home/translations/*"]
+    all: ["!**/*.py"]
```

Comment:
Why adding this? Translations will always be in .json, don't they?

Reply:
Yes they would, but I think it's better to set the explicit path to the `json` files so if we would to introduce any more `json` files and we change them, the GH won't mark them as "translation" label. For example, we have a file `hacs.json` but when that's modified we don't want to mark a PR with "translation" label :)

Reply:
I meant the `all` tag.